assignment.js

```diff
@@ -66,6 +72,18 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+let values = [3, 48, 29, 12, 18]
+let LargestNumber = values[0]
+
+for (i = 1; i < values.length; i++) {
```

Comment:
@justfinish nice use of a `for` loop here. Also good use of using your knowledge of arrays to use `i` as an index to implement the logic of keeping track of the largest number.